modules/Hoot/ui/modals/exportData.js

```diff
@@ -220,11 +220,19 @@ export default class ExportData {
             } )
             .catch( (err) => {
                 console.error(err);
+
                 let message = 'Error running export',
                     type = err.type,
                     keepOpen = true;
 
+                if (err.data.commandDetail.length > 0 && err.data.commandDetail[0].stderr !== '') {
+                    message = err.data.commandDetail[0].stderr;
```

Comment:
Do we want to overwrite the initial error message and not append to it instead?

Reply:
The only reason I did that for import multiple was that the user would want to know which file upload caused the error.  For single import and export I thought the error message from the hoot command was sufficient.

Reply:
good point